NoSQL_Schema

```diff
@@ -0,0 +1,54 @@
+
+db.addUser({
+	user: "super_admin",
+	password: password,
+	roles: [{"read","readWrite","userAdmin"}]
+})
+
+db.addUser({
+	user: "admin",
+	password: password,
+	roles: [{"read","readWrite"}]
+})
+
+db.addUser({
+	user: "super_admin",
+	password: password,
+	roles: ["read"]
+})
+
+
+user
+{
+	user_name: string,
+	password: password
+}
+
+profile
+{
+	full_name: name,
+	email_id: mail id of user,
+	user_name: username,
+	password: password
+}
+
+post
+{ 
+	post_id: id,
+	user_id: user id referenced,
+	post_title: title of post,
+	post_description: description_of_post,
+	post_by: post_action_by,
+	like: total_likes_count,
+	comments:
+	{
+		user: comment by,
+		comment: comments,
+		date: date of comment,
```

Comment:
- What date is this? Date of what? 
- The key name should be clear and precise eg: commented_at or created_at.